operator/cmd/seldon/cli/root.go

```diff
@@ -106,11 +106,16 @@ func GetCmd() *cobra.Command {
 	cmdConfigRemove := createConfigRemove()
 	cmdConfigList := createConfigList()
 
+	// Generic commands
+	cmdLoad := createLoad()
+	cmdUnload := createUnload()
+	cmdStatus := createStatus()
+
 	var rootCmd = &cobra.Command{Use: "seldon", SilenceErrors: false, SilenceUsage: true}
 
 	rootCmd.DisableAutoGenTag = true
 
-	rootCmd.AddCommand(cmdModel, cmdServer, cmdExperiment, cmdPipeline, cmdConfig)
+	rootCmd.AddCommand(cmdModel, cmdServer, cmdExperiment, cmdPipeline, cmdConfig, cmdLoad, cmdUnload, cmdStatus)
```

Comment:
just to keep consistency with previous usage (i.e. `seldon model load` and `seldon pipeline load`), an alternative is perhaps to use 
`seldon all load`? I am personally not entirely sure which approach is better so happy to leave it as is.

Reply:
seems a bit verbose - maybe will leave for now